lib/_http_server.js

```diff
@@ -82,6 +82,10 @@ const { observerCounts, constants } = internalBinding('performance');
 const { setTimeout, clearTimeout } = require('timers');
 const { NODE_PERFORMANCE_ENTRY_TYPE_HTTP } = constants;
 
+const dc = require('diagnostics_channel');
+const onRequestStartChannel = dc.channel('http.server.request.start');
+const onResponseFinishChannel = dc.channel('http.server.response.finish');
```

Comment:
Should these channels be documented at all, or are they not meant to be public API?

Reply:
They are intended to be documented eventually, but I'm not yet sure _how_ or _where_ they should be documented. Also, I think it might be best to hold off documenting them _for now_, until we decide on what the correlation (span) api, which I'm making another PR for and may influence the consuming convention of the request events.

cc @nodejs/documentation